src/Nest/DSL/UpdateSettingsDescriptor.cs

```diff
@@ -1,116 +1,154 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using Elasticsearch.Net;
 using Newtonsoft.Json;
 
 namespace Nest
-{
+{ 
+	public class SettingNames
```

Comment:
Yeah, that's true, looks like I need second coffee :)
I'll change name to `UpdatableSettings` later on.